src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ConstraintGammas.hpp

```diff
@@ -59,7 +59,7 @@ struct ConstraintGamma0Compute : ConstraintGamma0, db::ComputeTag {
           std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
           functions_of_time) noexcept {
     destructive_resize_components(gamma, get<0>(coords).size());
```

Comment:
Destructive resize to inside damping function, then test helper uses default constructor for the scalar to test that resizing happens correctly